### PR TITLE
Assert that sealengine is there.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: libethcore/SealEngine.cpp [hunk 1]
```diff
@@ -51,6 +51,7 @@ void SealEngineFace::verifyTransaction(ImportRequirements::value _ir, Transactio
 SealEngineFace* SealEngineRegistrar::create(ChainOperationParams const& _params)
 {
 	SealEngineFace* ret = create(_params.sealEngineName);
+	assert(!!ret && "Seal engine not found.");
```
Comment (bobsummerwill) [inline libethcore/SealEngine.cpp:54]: !!ret ?

Why not just ret?

Comment (chriseth) [inline libethcore/SealEngine.cpp:54]: yeah, might also work without `!!`. The `!!` is to force bool context, things like `assert(ret)` sometimes do not work on windows.

Comment (bobsummerwill) [inline libethcore/SealEngine.cpp:54]: Well, the most correct form would actually be "(ret != NULL)"

Comment (chriseth) [inline libethcore/SealEngine.cpp:54]: you mean `ret != nullptr` :-)

Comment (bobsummerwill) [inline libethcore/SealEngine.cpp:54]: I do as of C++11, yes :-)

**Pull request merged**